### ISSUE TITLE
MacOS is always pending

### ISSUE THREAD

**Issue by Allenxuxu** [actions/virtual-environments#382]
https://github.com/Allenxuxu/gev/runs/288928443

**Comment by bryanmacfarlane** [comment]
Not sure if this is still actionable but it sounds like it was waiting on a mac VM.  @Allenxuxu - still an issue?

**Comment by Allenxuxu** [comment]
Seems no problem now, let me close the issue first.
Thank you🙏.

**Comment by joehinkle11** [comment]
I have this same problem today. Is this something on GitHub’s end? Usually it will pend for 90 minutes before giving up

**Comment by maxim-lobanov** [comment]
@joehinkle11 , Yep, probably issue may come from our infrastructure.
Please create new issue if you see it one more time

**Comment by joehinkle11** [comment]
@maxim-lobanov Okay, yesterday it seemed to work, but if I breaks I'll make an issue
